submit.yml.erb

```diff
@@ -2,10 +2,10 @@
   cpus = bih_cpus.to_i
   mem = bih_mem.to_i
   time = bih_time
-  if time.include?("h")
-    formatted_time = "0-" + time.gsub("h", "")
+  if time.end_with?("h")
+    formatted_time = "0-" + time.delete_suffix("h")
   elsif time.include?("d")
```

Comment:
Hi Olli, thanks for following up. This should also be "time.end_with".

Reply:
great - should be updated